onboard/nucleo_bridge/Hardware.h

```diff
@@ -0,0 +1,102 @@
+#ifndef HARDWARE_H
+#define HARDWARE_H
+
+#include <algorithm>
+
+enum Type
```

Comment:
`Type` is a very generic term that doesn't help us understand what type it is describing. Can we create a better name?

Reply:
implemented